aicsimageio/aics_image.py

```diff
@@ -459,10 +459,29 @@ def get_channel_names(self, scene: int = 0) -> List[str]:
             List of strings representing the channel names.
         """
         try:
-            names = self.reader.get_channel_names(scene)
+            return self.reader.get_channel_names(scene)
         except AttributeError:
-            names = [str(i) for i in range(self.size_c)]
-        return names
+            return [str(i) for i in range(self.size_c)]
+
+    def get_physical_pixel_size(self, scene: int = 0) -> Tuple[float]:
+        """
+        Attempts to retrieve physical pixel size for the specified scene.
+        If none available, returns `1.0` for each spatial dimension.
+
+        Parameters
+        ----------
+        scene: int
+            The index of the scene for which to return physical pixel sizes.
+
+        Returns
+        -------
+        sizes: Tuple[float]
+            Tuple of floats representing the pixel sizes for X, Y, Z, in that order.
+        """
+        try:
+            return self.reader.get_physical_pixel_size(scene)
+        except AttributeError:
+            return (1.0, 1.0, 1.0)
```

Comment:
If we're concerned about being consistent, why not change this function at the reader level?

Reply:
I agree that the base class reader might as well just return (1,1,1).

Reply:
I would say do both. If the reader doesn't implement the function then we still default to this, but in each reader it should have returns for if available or not.

Reply:
If it's on the base reader than any reader that doesn't implement the function will use the base method's class, so I don't think it needs to be in both places. Then the OmeTiffReader would just replace the base method for this.

Reply:
Ohhhh I see what you are saying. Smart!